src/components/media-player/main-track-editor.js

```diff
@@ -65,13 +65,16 @@ function MainTrackEditor({ content, onContentChanged }) {
 
   const handleSourceTypeChange = event => {
     const { value } = event.target;
-    changeContent({
+    const newContent = {
       sourceType: value,
       sourceUrl: '',
       range: [0, 1],
```

Comment:
BTW I discovered this wrongly named property. It should be `playbackRange`. It's fixed in my PR (https://github.com/educandu/educandu/pull/602), but you could fix it here to to avoid conflicts.

Reply:
Huh.. Ok, fixed it in this PR too.